MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1743,6 +1743,13 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
 
                 filters.Add(subParam);
 
+		// Ensure proper filters are passed to ffmpeg in case of hardware acceleration via VA-API
```

Comment:
indentation seems off, you seem to be using tabs instead of spaces

Reply:
This was done hackily on the machine where the serer runs. Will fix.